server/src/domain/system-config/system-config.util.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { ServerVersion } from '..';
```

Comment:
What do you think about adding the `compareVersions` method to the `ServerVersion` class instead, extracting it into a separate file `domain/server-info/server-version.ts` and having a separate `server-version.spec.ts` where all these tests live?